crates/matrix-sdk/src/room/mod.rs

```diff
@@ -315,6 +316,24 @@ impl Room {
         self.client.subscribe_to_room_updates(self.room_id())
     }
 
+    /// Subscribe to typing notifications for this room.
+    ///
+    /// The returned receiver will receive a new vector of user IDs for each
+    /// sync response that contains 'm.typing' event.
+    pub fn subscribe_to_typing_notifications(
+        &self,
+    ) -> (EventHandlerDropGuard, broadcast::Receiver<Vec<OwnedUserId>>) {
+        let (sender, receiver) = broadcast::channel(16);
+        let typing_event_handler_handle = self.client.add_room_event_handler(self.room_id(), {
+            let sender = sender.clone();
+            move |ev: SyncTypingEvent| async move {
```

Comment:
Cloning `sender` isn't necessary as it is moved inside the closure.

Also, please rename `ev` to `event`.

```suggestion
            move |event: SyncTypingEvent| async move {
```